src/middlewares/phpAuth.js

```diff
@@ -13,7 +13,8 @@ const phpAuth = async function (req, res, next) {
   console.log(JSON.stringify(req.cookies.PHPSESSID));
   const response = await axios.get('https://www.motorsingh.com/user/validate', {
     // headers: { Cookie: `PHPSESSID=${req.cookies.PHPSESSID};` },
-    headers: { Cookie: `PHPSESSID=7e952iigfbbkvle1v0j61tn8c3` },
+    // headers: { Cookie: `PHPSESSID=7e952iigfbbkvle1v0j61tn8c3` },
+    headers: { Cookie: `PHPSESSID=0pt78bg40irspangui51l1nfc6` },
```

Comment:
Plz revert this change.
IT should be 
    ``headers: { Cookie: `PHPSESSID=${req.cookies.PHPSESSID};` },`